java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/DeployCommand.java

```diff
@@ -380,6 +393,13 @@ public void accept(String s, String hash) {
                     deploymentPath);
             deployer.deploy();
         }
+        if ((isNotEmpty(gitURL) || isNotEmpty(gitIdentity)) && gitToken.isPresent()) {
+            var git = Git.builder(gitURL, gitIdentity, gitToken.get());
+            String name = scmUri.endsWith(".git") ? scmUri.substring(0, scmUri.length() - 4) : scmUri;
+            name = name.substring(name.lastIndexOf("/") + 1);
```

Comment:
I think we also need to work the rest of the URI into the name, by transforming it into a 'safe' name for a git repo.

e.g. something like github.com/foo/bar would end up as github.com/<user>/foo-bar. 

Possibly we also need to think about what if github and gitlab have the same org/repo combo but different, but that seems unlikely, unless a project has changed from one to the other.

Reply:
Hmm. Good point, though even if they both have the same combination we should be deploying to a new user/org so it shouldn't matter. But I think the originating user/org would be required as otherwise we might have foo/bar and foo/foo so without the user/org they would clash.